spec/support/corpus.rb

```diff
@@ -124,6 +116,42 @@ def checkout
 
     private
 
+      MissingExpectedException = Class.new(StandardError)
```

Comment:
Woops I think this might be a holdover actually from the initial implementation. Now I just re-raise